.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+dist: trusty
+sudo: false
+
+git:
+    depth: 1
+
+php:
+  - 7.1
+  - 7.2
+
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+        - vendor
+        - .phpunit
+
+env:
+    global:
+        - SYMFONY_PHPUNIT_DIR=.phpunit
+
+install:
+    - composer --prefer-source install
```

Comment:
`--prefer-source` must be replaced by `--prefer-dist` (otherwise persisting the composer cache is useless)